package.json

```diff
@@ -132,13 +133,15 @@
     "preset": "react-native",
     "roots": [
       "<rootDir>/node_modules/react-native/",
-      "<rootDir>/lib/",
+      "<rootDir>/src/",
```

Comment:
Please also change this:

https://github.com/facebook/prepack/blob/afc77da9a0ec97fc1dec171fc2244326a7bfde54/test/react/setupReactTests.js#L14

and this:

https://github.com/facebook/prepack/blob/afc77da9a0ec97fc1dec171fc2244326a7bfde54/test/react/setupReactTests.js#L24

to use `src`.

Please verify that changing files in `src` re-runs the tests in `yarn test-react --watch` mode even if you don't run `yarn watch`.